packages/phoenix-ng/projects/phoenix-app/src/app/sections/atlas/atlas.component.spec.ts

```diff
@@ -26,10 +46,4 @@ describe('AtlasComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
-
-  // Test if three.js is initialized
-  it('should initialize three.js canvas', () => {
```

Comment:
It would have been possible with Jasmine (karma) and Testbed, but IMHO, I don't think we can test Angular services without mocking wrt Jest (jsdom) as we'll get errors. There didn't seem to be another way as I've [seen many people mocking them](https://stackoverflow.com/q/71809333/13616209) in their tests.